ui/apps/platform/cypress/integration/vulnmanagement/reporting/reporting.helpers.js

```diff
@@ -110,10 +112,31 @@ const routeMatcherMapToCreate = {
     },
 };
 
-export function visitVulnerabilityReportingToCreate(staticResponseMap) {
-    visit(`${basePath}?action=create`, routeMatcherMapToCreate, staticResponseMap);
+const routeMatcherMapToCreateWithCollections = {
+    [collectionsAlias]: {
+        method: 'GET',
+        url: '/v1/collections*',
+    },
+    [notifiersAlias]: {
+        method: 'GET',
+        url: '/v1/notifiers',
+    },
+};
+
+export function visitVulnerabilityReportingToCreate(staticResponseMap, isCollectionsEnabled) {
```

Comment:
I went with an additional parameter based on the feature flag here since I think that will make these easier to track down and modify once the flag is deleted, but I could create separate helper functions as well if we would prefer.

Reply:
This is good.